pages/pictures.html

```diff
@@ -2,19 +2,20 @@
 
 <head>
     <title>Pictures</title>
+    <link rel="stylesheet" href="../css/style.css">
 </head>
 
 <body>
     <ul>
-        <li><a href="../index.html">Home</a></li>
-        <li><a href="pictures.html">Gallery</a></li>
-        <li><a href="chloro.html">Chlorophytum data</a></li>
-        <li><a href="sanseviera.html">Sansevieria data</a></li>
-        <li><a href="Epipremnum.html">Epipremnum data</a></li>
+        <li class="page-link"><a href="../index.html">Home</a></li>
+        <li class="page-link"><a href="pictures.html">Gallery</a></li>
+        <li class="page-link"><a href="chloro.html">Chlorophytum data</a></li>
+        <li class="page-link"><a href="sanseviera.html">Sansevieria data</a></li>
+        <li class="page-link"><a href="Epipremnum.html">Epipremnum data</a></li>
     </ul>
-    <a href="../pages/sanseviera.html"><img src="../images/sansevieria.jpg" alt="Plants" style="width: 400px;;"></a>
-    <a href="../pages/chloro.html"><img src="../images/Chlorophytum.jpg" alt="Plants" style="width: 400px;;"></a>
-    <a href="../pages/Epipremnum.html"><img src="../images/Epipremnum-aureum.jpg" , style="width: 400px;;"></a>
+    <a href="../pages/sanseviera.html"><img src="../images/sansevieria.jpg" alt="Plants"></a>
+    <a href="../pages/chloro.html"><img src="../images/Chlorophytum.jpg" alt="Plants"></a>
+    <a href="../pages/Epipremnum.html"><img src="../images/Epipremnum-aureum.jpg" alt="Plants"></a>
```

Comment:
why does only this img have `alt`? also - the `alt` is general and doesn't describe this image specifically